scripts/base_train.py

```diff
@@ -58,6 +58,8 @@
 core_metric_every = 2000 # every how many steps to evaluate the core metric (-1 = disable)
 core_metric_max_per_task = 500 # examples per task in estimating the core metric
 sample_every = 2000 # every how many steps to sample from the model
+# Logging
+promised_flops_per_gpu = 989e12 # Only used for logging. Set it to "without sparsity, bfloat16 flops for a single gpu", to log correct `mfu`. Default is set for H100 SXM, bfloat16, without 2:4 sparsity.
```

Comment:
No need to repeat "logging" twice + some little tweaks to the description
```suggestion
promised_flops_per_gpu = 989e12 # To log the correct `mfu`, set this to the value of your GPU, "without sparsity, bfloat16 flops for a single gpu". Default is set for H100 SXM, bfloat16, without 2:4 sparsity.
```